tensorflow/lite/kernels/slice.cc

```diff
@@ -117,7 +117,8 @@ TfLiteStatus Prepare(TfLiteContext* context, TfLiteNode* node) {
                  begin->type == kTfLiteInt32 || begin->type == kTfLiteInt64);
   TF_LITE_ENSURE(context,
                  size->type == kTfLiteInt32 || size->type == kTfLiteInt64);
-  TF_LITE_ENSURE(context, NumDimensions(begin) == NumDimensions(size) == 1);
+  TF_LITE_ENSURE(context, NumDimensions(begin) == 1);
+  TF_LITE_ENSURE(context, NumDimensions(size) == 1);
```

Comment:
You should use TF_LITE_ENSURE_EQ here (see examples above). Unlike TF_LITE_ENSURE, this will print the number of dimensions if the check fails.

Reply:
@skye , thanks for the comments, i have updated the code as per the comment, kindly approve the PR.